test/IISIntegration.FunctionalTests/HelloWorldTest.cs

```diff
@@ -20,31 +19,23 @@ public HelloWorldTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        [ConditionalTheory]
-        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
```

Comment:
Are you going to run these on Linux/Mac now? That won't go well. You could promote this condition to the assembly level like we did for HttpSysServer: https://github.com/aspnet/HttpSysServer/blob/dev/test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/Properties/AssemblyInfo.cs